platform_channels/lib/src/method_channel_demo.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:platform_channels/src/counter_method_channel.dart';
+
+/// The widget demonstrates how to use [MethodChannel] to invoke platform methods.
+/// It has two [FloatingActionButton]s and a [Text] widget to display the value of
+/// [count].
+class MethodChannelDemo extends StatefulWidget {
+  @override
+  _MethodChannelDemoState createState() => _MethodChannelDemoState();
+}
+
+class _MethodChannelDemoState extends State<MethodChannelDemo> {
+  int count;
+  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
```

Comment:
You can get around the need for this key by wrapping the body of your scaffold in a builder widget, which will provide a new BuildContext below the Scaffold.

Reply:
I did the same thing. First, I was using `Builder` to have a different context, thinking that GlobalKey is a little more expensive, but I switch to GlobalKey so that I have to pass less arguments. 

I'll again use the `Builder` instead of `GlobalKey`.